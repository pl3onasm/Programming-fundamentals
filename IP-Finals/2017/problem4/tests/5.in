Beauty is in the eye of the beholder.