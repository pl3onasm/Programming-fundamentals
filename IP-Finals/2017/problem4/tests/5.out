B##ytheitehoash#elu#eo#dti#fbeyne#er
